### DISCUSSION TITLE
Is marker-url supported for geojson in conjunction with perPropertyStyles

### DISCUSSION THREAD

**Discussion by lstillwe** [TerriaJS/terriajs#6829] [Q&A]
I have a perPropertyStyles set like this and wonder if something like this should work in TerriaJS V8?

```
"perPropertyStyles": [
       {
           "properties": { "storm_type": "TD" },
             "style": {
                "marker-url": "https://some.path.org/dep.png"
           },
       },
       .
       .
       .
]
```
Thanks very much!

**Comment by sixlighthouses** [comment]
Please take a look at the docs here [GeoJsonCatalogItem](https://docs.terria.io/guide/connecting-to-data/catalog-type-details/geojson/#pointsymboltraits), in you example from above it would be 
```
"perPropertyStyles": [
       {
           "properties": { "storm_type": "TD" },
             "style": {
                "marker": "https://some.path.org/dep.png"
           },
       },
       .
       .
       .
]
```

**Reply by lstillwe** [reply]
Thank you for the suggestion. I did try just marker and that didn't work either.
I have modified a couple of lines of code in GeojsonMixin.ts (loadGeoJsonDataSource function) and now this works!
If anyone else it interested, I will be happy to share fix.

**Reply by imakihi** [reply]
Hi there. I encounterd the same issue. How did you make it work?
Thank you.
